ldk/javascript/examples/self-test-loop/src/tests/document/index.tsx

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable no-async-promise-executor */
+import { React, ReactWhisper, document, network, whisper } from '@oliveai/ldk';
+import { Workbook, PDFContentType, PDFOutput } from '@oliveai/ldk/dist/document/types';
+
+export const testDocumentEncodeAndDecode = (): Promise<boolean> =>
+  new Promise(async (resolve, reject) => {
+    const workbook: Workbook = {
+      worksheets: [
+        {
+          hidden: false,
+          hiddenColumns: [],
+          hiddenRows: [],
+          name: 'name',
+          rows: [{ cells: [{ value: 'value' }] }],
+        },
+      ],
+    };
+    setTimeout(() => {
+      reject(new Error('XLSX encode did not finish in the appropriate time span.'));
+    }, 5000);
+
+    try {
+      const uint8ArrayData = await document.xlsxEncode(workbook);
+      const actual = await document.xlsxDecode(uint8ArrayData);
+      const cellData = actual.worksheets[0].rows[0].cells[0].value;
```

Comment:
Will this always reliably access each key here?

Reply:
In the case of this unit test, yep!